StudyHub/StudyHub.BLL/Services/Assignments/VariantService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using StudyHub.BLL.Services.Interfaces.Assignment;
+using StudyHub.Common.DTO.TaskVariant;
+using StudyHub.Common.Exceptions;
+using StudyHub.DAL.Repositories.Interfaces;
+using StudyHub.Entities;
+
+namespace StudyHub.BLL.Services.Assignments;
+
+public class VariantService : IVariantService
+{
+    private readonly IMapper _mapper;   
+    private readonly IRepository<TaskVariant> _taskVariantRepository;
+    private readonly IRepository<AssignmentTask> _assignmentTaskRepository;
+
+    public VariantService(IMapper mapper, 
+        IRepository<TaskVariant> taskVariantRepository, 
+        IRepository<AssignmentTask> assignmentTaskRepository)
+    {
+        _mapper = mapper;
+        _taskVariantRepository = taskVariantRepository;
+        _assignmentTaskRepository = assignmentTaskRepository;
+    }
+
+    public async Task<TaskVariantDTO> CreateTaskVariantAsync(Guid assignmentTaskId, CreateTaskVariantDTO taskVariant)
```

Comment:
we said that options should be added as a list

Reply:
wdum? It`s a variant endpoint, not option, or I dont get something?